huaweicloud/services/acceptance/cfw/data_source_huaweicloud_cfw_firewalls_test.go

```diff
@@ -0,0 +1,38 @@
+package cfw
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDatasourceFirewalls_basic(t *testing.T) {
+	rName := "data.huaweicloud_cfw_firewalls.test"
+	dc := acceptance.InitDataSourceCheck(rName)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheckCfw(t) },
```

Comment:
acceptance.TestAccPreCheck(t) is missing

Reply:
fixed